Makefile

```diff
@@ -109,13 +109,13 @@ skaffold-dev-gcpmanager: protoc skaffold protogen render-skaffold-manifests ## R
 build: manifests generate fmt vet ## Build manager binary.
 	go build -o bin/manager cmd/controllermanager/main.go
 
-.PHONY: dev
+.PHONY: releases
 dev: manifests kustomize install-crds
```

Comment:
I don't quite understand this - why have a PHONY (i.e., [a phony make target that doesn't correspond to a file that changes](https://stackoverflow.com/a/2145605)) that doesn't match the following line for the target action? I guess I've never seen this pattern where they don't match.

Reply:
I have no idea where this came from

Reply:
Changed